chia/util/network.py

```diff
@@ -69,7 +36,8 @@ async def create(
         hostname: str,
         port: uint16,
         routes: Iterable[web.RouteDef] = (),
-        max_request_body_size: int = 1024**2,  # Default `client_max_size` from web.Application
+        max_request_body_size: int = 1024
+        ** 2,  # Default `client_max_size` from web.Application
```

Comment:
I would have expected you need parentheses when breaking up an expression like this